src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -0,0 +1,98 @@
+import { NavLink } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import classNames from 'classnames';
+
+import { changeBurgerState } from '@/features/burgermenu/burgerSlice';
+
+import fav from '../../../src/assets/images/icons/favorites-blank.svg';
+import cart from '../../../src/assets/images/icons/shopping-bag-blank.svg';
+
+import { Paths } from '@/enums';
+
+import styles from './styles.module.scss';
+
+const getActiveNavLinkOnBurger = ({ isActive }: { isActive: boolean }) => {
+  return classNames(styles.burger__nav__links, {
+    [styles.burger__nav__links__active]: isActive,
+  });
+};
+
+const getActiveNavLinkOnBurgerFooter = ({
+  isActive,
+}: {
+  isActive: boolean;
+}) => {
+  return classNames(styles.burger__footer__links, {
+    [styles.burger__footer__links__active]: isActive,
+  });
+};
+
+export const BurgerMenu = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <menu className={styles.burger}>
+      <div className={styles.container}>
+        <nav className={styles.burger__nav}>
```

Comment:
Even if it's not particularly clear how to reuse the Navigation component it can still be done via a `map` and `import { HeaderNavigationLinks } from '@/constants';`